portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -106,9 +106,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<BlogMessage> BlogMessages = getBlogsFromDatastore();
 
     // Create BlogHashMap Object and put BlogMessages in the map.
+    
```

Comment:
Can we also update the comment in line 108 to give more context here since you added more logic here. The original comment is kind of vague at first glance.